core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -1014,9 +1014,9 @@ default TimeType getTimeType(String tableName) throws TableNotFoundException {
   }
 
   /**
-   * Initiates setting a table to ondemand state, but does not wait for action to complete
+   * Initiates setting a table to on-demand state, but does not wait for action to complete
```

Comment:
Small grammar things here. This could read better if it's setting a table to 'an' or maybe 'the' on-demand state ('an' for better grammar or 'the' to specify that it is a unique table state in accumulo).